src/tasks/installers/installer.ts

```diff
@@ -91,10 +95,12 @@ export class InstallerTasks {
 
     if (flags.installer === 'operator') {
       title = '🏃‍  Running the Eclipse Che operator'
-      task = () => operatorTasks.deployTasks(flags, command)
```

Comment:
I don't understand these changes.

Reply:
To have a subtasks list we need to wrap the tasks in Listr object. It was originally done in the get tasks function. But I want to reuse the tasks and getting them already wrapped is not an option. So, I made the get tasks return just a task list and wrap it, when/if needed. We already have such approach in a some places and to me it seems correct.